mobile/app/(app)/change-password.tsx

```diff
@@ -89,7 +89,7 @@ const ChangePassword = () => {
   };
 
   if (successfullyChanged) {
-    return <ChangePasswordConfirmation />;
+    return <ChangePasswordConfirmation emailConfirmation={false} />;
```

Comment:
given that we are using the ChangePasswordConfirmation component both for confirming a password change or for the email sent step in forgot password, maybe we should also pass as a prop the action text on the back button (when we're in the forgot password case, the button should display `Back to log in`, whereas on confirming the password change we need `Back to app` )

Reply:
discussed offline